tools/format.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 import os
-from third_party import third_party_path, fix_symlinks, google_env
+from third_party import third_party_path, fix_symlinks, google_env, clang_format_path
```

Comment:
This line is over 80 characters yet yapf didn't format it. It's fine for now. But I think yapf needs to be configured or replaced.